src/lib/userRepository.ts

```diff
@@ -296,7 +296,24 @@ class UserRepository {
 		});
 	}
 
-	public async updateUserPswd(userId: string, password: string) {
+
+	public updateUserEmail(userId: string, email: string) : RunResult {
+		try {
+			const sql = this.db.prepare<[string, string]>(
```

Comment:
This statement does not return anything so I would just return promise void
If you want you can return the amount of rows edited but I don't think it's necessary